deploy/wiremock/wiremock-deployment.yaml

```diff
@@ -16,6 +16,7 @@ spec:
       containers:
         - name: wiremock
           image: quay.io/ocpmetal/wiremock
```

Comment:
Just for reference, we should figure out how this was built and if we should update it or move it to `edge-infrastructure` I don't think we do anything with `ocpmetal` anymore.

cc @gamli75 what happened with that org? Do we even own it still?

Reply:
the org still exists but I agree that we should move it